javascript/packages/orchestrator/src/configGenerator.ts

```diff
@@ -364,6 +364,7 @@ export async function generateNetworkSpec(
           parachain.register_para === undefined
             ? true
             : parachain.register_para, // register by default
+        onboardAsParachain: parachain.onboard_as_parachain || true, // onboard by default
```

Comment:
It will never be false. 
```suggestion
        onboardAsParachain: parachain.onboard_as_parachain !== undefined ? parachain.onboard_as_parachain : true, // onboard by default
```